elytron/src/main/java/org/wildfly/extension/elytron/CredentialStoreResourceDefinition.java

```diff
@@ -192,7 +216,17 @@ private CredentialStoreAddHandler() {
         protected void performRuntime(OperationContext context, ModelNode operation, Resource resource) throws OperationFailedException {
 
             ModelNode model = resource.getModel();
-            String uri = asStringIfDefined(context, URI, model);
+            String location = asStringIfDefined(context, LOCATION, model);
+            boolean modifiable = asBooleanIfDefined(context, MODIFIABLE, model);
+            boolean create = asBooleanIfDefined(context, CREATE, model);
```

Comment:
If you know MODIFIABLE has a default value there is no reason to do anything other than

boolean modifiable = MODIFIABLE.resolveModelAttribute(context, model).asBoolean();

Same for CREATE.

The contract of resolveModeAttribute is to return any default if the value in the model is undefined.